ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/ScanConfigActionDropdown.tsx

```diff
@@ -104,6 +107,20 @@ function ScanConfigActionDropdown({
         /* eslint-enable no-nested-ternary */
     }
 
+    if (isComplianceReportingEnabled && isReportJobsEnabled) {
```

Comment:
Aha, after you merge work in progress, I will actively pursue deleting the previous feature flag.

Reply:
I can do that too. It shouldn't be a big effort and I'm in the code anyways 👍🏼